projects/cdk/components/dialog-host/dialog-host.template.html

```diff
@@ -12,5 +12,10 @@
         [context]="item"
         [@tuiParentAnimation]
     ></section>
-    <div class="overlay" [class.overlay_visible]="dialogs.length"></div>
+    <div
+        class="overlay"
+        (window:popstate)="closeLast(dialogs)"
+        (transitionstart)="onDialog($event ,!!dialogs.length)"
```

Comment:
Wrong space around comma. Strange, I thought prettier would fix this 🤔

Reply:
fixed